client/eth/client.go

```diff
@@ -47,7 +47,10 @@ type Reader interface {
 	) (tx *ethcoretypes.Transaction, isPending bool, err error)
 	TxPoolContent(
 		ctx context.Context) (
-		map[string]map[string]map[string]*ethcoretypes.Transaction, error)
+		map[string]map[common.Address]map[string]*ethcoretypes.Transaction, error)
+	TxPoolInspect(
+		ctx context.Context,
+	) (map[string]map[common.Address]map[string]string, error)
```

Comment:
since this is the interface exported to user, can you comment above each of these funcs what the return type looks like (what each of the keys correspond to) and maybe past an example json response